api/descriptions/policies/vulnerability_images_put.md

```diff
@@ -1,33 +1,108 @@
-Updates all rules in your vulnerability policy for container images in a single shot.
-Updating all rules at the same time makes it possible to maintain strict ordering between rules.
+Simultaneously updates all container image rules for your vulnerability policy. The simultaneous rule update enables maintenance of strict rule ordering.
 
-The procedure to add, edit, or remove vulnerability rules is:
+### cURL Request
 
-1. Get all vulnerability rules using the GET endpoint.
+The following cURL command triggers Console to update rules in your vulnerability policy for container images.
 
-  The following curl command uses basic auth to retrieve a list of all vulnerability rules, pretty-print the JSON response, and save the results to a file.
+```bash
+$ curl -k \
```

Comment:
@KICKdesigns 

I'd like to distill our example curl commands down to as simple as possible. Unfortunately, our API JSON doesn't specify what params are optional and which are mandatory. I'd like the curl command to be as simple as possible as sort of a rough guide for what's optional and mandatory. From there, customers can build on the example curl commands to customize it for their needs.

This will require a bit of experimentation on your end to figure it out

For example, the following things can be simplified or removed:

* `collections` object (based on my feedback in this PR #292)
* `modified` - not needed, the system will calculate it for you.
* `owner` - not needed, implied by the user that makes the curl command
* `previousName` - not needed, internal detail managed by the system
* `action` - I think this is nothing but a placeholder, please experiment and validate.
* etc, etc

Reply:
@iansk It seems that for some of the endpoints only the name is required. Do you still want to include info such as a shortened collections?

Reply:
@KICKdesigns Clarifying question: Do you mean when creating a rule, only `name` is required? If so, let's make the cURL command a little more meaningful to reflect what's minimally required in the real world. In particular, make sure the rule has:

* A name
* A scope (with one more more collections)
* Alert/block thresholds.

Everything else, I think they can figure out.

Reply:
@iansk Yes that’s what I mean. Will do on the param list to include